lib/connection/spawnInterruptibleJulia.ps1

```diff
@@ -1,11 +1,11 @@
 param(
-	[Int32] $port, 
-	[string] $jlpath, 
-	[string] $jloptions
+	[Int32] $port,
+	[string] $jlpath,
+	[string] $boot
 )
 
 # start Julia
-$proc = Start-Process $jlpath "$jloptions -e `"import Atom; @sync Atom.connect($port)`"" -NoNewWindow -PassThru
+$proc = Start-Process $jlpath @($boot, $port) -NoNewWindow -PassThru
```

Comment:
Yeah, I'll try hacking something together this evening -- Windows is a nightmare when it comes to doing something like that.